packages/rxjs-traces/src/devtools.ts

```diff
@@ -119,3 +123,43 @@ function prepareForTransmit<T>(
       return value;
   }
 }
+
+function publishReplayOnce<T>(source: Observable<T>) {
```

Comment:
Another interesting operator: Acts like a publishReplay, but only to the first subscriber.

This means that when you call `.connect()`, it will subscribe and start accumulating values, and as soon as there's a subscriber it will replay every value sent, and then it will just proxy all the new messages to that subscriber.

The effect is similar to `pipe(publishReplay(), share())`, but if there's a subscriber it won't keep references to each value.

Not for public use - it's missing the multicast bit 😅 - In here it's used with just one subscriber.